constellation/src/storage/rocks_store.rs

```diff
@@ -270,10 +292,20 @@ impl RocksStorage {
             // the reverse links:
             ColumnFamilyDescriptor::new(TARGET_LINKERS_CF, {
                 let mut opts = rocks_opts_base();
+
                 opts.set_merge_operator_associative(
                     "merge_op_extend_did_ids",
                     Self::merge_op_extend_did_ids,
                 );
+
+                let mut bopts = rocksdb::BlockBasedOptions::default();
+                // bopts.set_block_size(/* 16 KiB */ 16 * 1_024);
+                bopts.set_pin_top_level_index_and_filter(true);
+                bopts.set_index_type(rocksdb::BlockBasedIndexType::TwoLevelIndexSearch);
+                bopts.set_partition_filters(true);
```

Comment:
I hope my intuition was right and `TARGET_LINKERS_CF` is the largest CF. The hope is to reduce cache movement when paging in and out the index/filter blocks.